classes/event/entry_updated.php

```diff
@@ -0,0 +1,34 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace mod_cpdlogbook\event;
+
+class entry_updated extends entry_created {
+
+    public function init() {
+        parent::init();
+        $this->data['crud'] = 'u';
+    }
+
+    public function get_description() {
```

Comment:
Can you please add the url so you can deep link from the event log into the entry, do this for update and create